examples/shopping-cart/src/reducers/products.js

```diff
@@ -49,10 +49,8 @@ export default combineReducers({
   visibleIds
 })
 
-export function getProduct(state, id) {
-  return state.byId[id]
-}
+export const getProduct = (state, id) => state.byId[id]
 
-export function getVisibleProducts(state) {
-  return state.visibleIds.map(id => getProduct(state, id))
-}
+export const getVisibleProducts = state => state.visibleIds.map(id =>
```

Comment:
Thanks for the styling advices, I'll do a pass on all of my PRs to conform with the comments you made.